bundle.js

```diff
@@ -0,0 +1,714 @@
+    'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\',
+    'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', "'",
+    'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/',
+];
+const keyLayout1_eng_up = [
+    '~', '!', '@', '#', '$', '%', '^', '&', '*', '(', ')', '_', '+',
+    'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|',
+    'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ':', '"',
+    'Z', 'X', 'C', 'V', 'B', 'N', 'M', '<', '>', '?',
+];
+const small_rus = [
+    '"', '№', ';', '%',
+    '\\',
+];
+const small_eng = [
+    '@', '#', '$', '^', '&', '|', '"', '<', '>', '?'
+];
+const row_1 = document.createElement('div');
+const row_2 = document.createElement('div');
+const row_3 = document.createElement('div');
+const row_4 = document.createElement('div');
+const row_5 = document.createElement('div');
+row_1.classList.add('row');
+KEYBORD.appendChild(row_1);
+row_2.classList.add('row');
+KEYBORD.appendChild(row_2);
+row_3.classList.add('row');
+KEYBORD.appendChild(row_3);
+row_4.classList.add('row');
+KEYBORD.appendChild(row_4);
+row_5.classList.add('row');
+KEYBORD.appendChild(row_5);
+
+for (let i = 0; i < 13; i++) {
+    const number_button = document.createElement('div');
+    number_button.classList.add('simple_button');
+    const number_button_span_rus = document.createElement('span');
+    number_button_span_rus.classList.add('rus');
+    const number_button_span_eng = document.createElement('span');
+    number_button_span_eng.classList.add('eng');
+
+    const number_button_span_rus_down = document.createElement('span');
+    number_button_span_rus_down.classList.add('down');
+    const number_button_span_rus_up = document.createElement('span');
+    number_button_span_rus_up.classList.add('up');
+    number_button_span_rus_down.innerText = keyLayout1_rus_down[i];
+    number_button_span_rus_up.innerText = keyLayout1_rus_up[i];
+    number_button_span_rus.appendChild(number_button_span_rus_down);
+    number_button_span_rus.appendChild(number_button_span_rus_up);
+    number_button.appendChild(number_button_span_rus);
+
+    const number_button_span_eng_down = document.createElement('span');
+    number_button_span_eng_down.classList.add('down');
+    const number_button_span_eng_up = document.createElement('span');
+    number_button_span_eng_up.classList.add('up');
+    number_button_span_eng_down.innerText = keyLayout1_eng_down[i];
+    number_button_span_eng_up.innerText = keyLayout1_eng_up[i];
+    number_button_span_eng.appendChild(number_button_span_eng_down);
+    number_button_span_eng.appendChild(number_button_span_eng_up);
+    number_button.appendChild(number_button_span_eng);
+
+    row_1.appendChild(number_button);
+}
+const BACKSPACE = document.createElement('div');
+const BACKSPACE_SPAN_ENG = document.createElement('span');
+BACKSPACE_SPAN_ENG.innerText = "BackSpace";
+BACKSPACE.appendChild(BACKSPACE_SPAN_ENG);
+BACKSPACE.classList.add('simple_button');
```

Comment:
You can write it to one line:
BACKSPACE.classList.add('simple_button', 'Backspace', 'tool');